tests/bsim/bluetooth/mesh/tests_scripts/persistence/prov_with_stuck_key.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright 2025 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test checks that mesh provisioning data is saved and loaded correctly
+# if a key with ID from mesh PSA key ID range exists in the PSA ITS,
+# it is destroyed and the correct key is used instead.
+#
+# Test must be added in pair and in sequence.
+# First test: saves data; second test: verifies it.
+#
+# Test procedure:
```

Comment:
It is also bit new that we are now documenting the test procedure within the ".sh" file. This is not bad, but perhaps we should do this all tests eventually whenever we touch them.